src/ssl.c

```diff
@@ -3612,8 +3612,10 @@ int wolfSSL_SetVersion(WOLFSSL* ssl, int version)
 /* Make a work from the front of random hash */
 static INLINE word32 MakeWordFromHash(const byte* hashID)
 {
-    return (hashID[0] << 24) | (hashID[1] << 16) | (hashID[2] <<  8) |
-            hashID[3];
+    return (((word32)hashID[0] << 24) |
```

Comment:
Won't this crash platforms with strong alignment requirements?

How is this different than ato32()?

Reply:
We are casting the value of hashID to word32 then left shift. I don't see the alignment issue here. Maybe it would be more clear if we had an additional set of parens such as `(((word32)hashID[0]) << 24)`?